content/chapter-sources/webhook/_index.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Webhook
+linktitle: "webhook() source"
+weight: 6000
+driver: "webhook(), webhook-json()"
+short_description: "Receive logs via a HTTP webhook"
+---
+<!-- This file is under the copyright of Axoflow, and licensed under Apache License 2.0, except for using the Axoflow and AxoSyslog trademarks. -->
+
+Starting with version 4.8.0, {{% param "product_name" %}} can collect logs via a webhook using the `webhook()` and `webhook-json()` sources. The `webhook-json()` source automatically parses the payload using the [`json-parser()`]({{< relref "/chapter-parsers/json-parser/_index.md" >}}).
+
+Example minimal config:
+
+```shell
+source s_webhook {
+webhook-json(
+    port(8181)
+    paths(["/events","/events/(?P<HOST>.*)"])
+    );
+};
+```
+
+To test the source, you can use `curl` for example, on the host where {{< product >}} is running:
+
+```shell
+curl -X POST --data "{'MESSAGE':'message-value'}" http://127.0.0.1:8181/events
+```
+
+This driver is actually a reusable configuration snippet based on a [custom Python source]({{< relref "/chapter-sources/python-source/_index.md" >}}). For details on using or writing such configuration snippets, see {{% xref "/chapter-configuration-file/large-configs/config-blocks/_index.md" %}}. You can find the source of this configuration snippet on [GitHub](https://github.com/axoflow/axosyslog/blob/main/modules/python-modules/syslogng/modules/webhook/scl/webhook.conf).
+
+## HTTPS webhook
+
+To receive data using HTTPS, configure a certificate and a private key for the webhook using the `tls_cert_file` and `tls_key_file` options.
+
+To verify the certificate of the clients sending data to the webhook, set `tls_peer_verify(yes)`, and use the following options:
+
+- `tls_use_system_cert_store(no)`, or
+- `tls_ca_file("")`, or
+- `tls_ca_dir("")`
+
+## Options
+
+## auth_token()
+
+<!-- FIXME -->
```

Comment:
It can be used as an additional layer of authentication. It is not recommended to use this under plain HTTP.

```
auth-token("Basic dGVzdF9zZWNyZXRfYXV0aF90b2tlbg==")
auth-token("Bearer <token>")
```

When this is set, only requests containing the `Authorization` header (the exact same value that is specified) will be accepted. Every other requests will be rejected with 403.